### ISSUE TITLE
Calulate end of task according to ressource rate

### ISSUE THREAD

**Issue by MisterFred** [bardsoftware/ganttproject#1214]
Hi!
I am running three tasks in parallel.
Imagine I need 3 days of work to complete the first task.
But I allocate only 50% of my time for this.
Thus I would like this task to end in 6 days of work.
Same thing for task 2 is a 2 days of work with a 20% load allocation. It should finish in 10 days.
And the 3rd task is a 4 days of work load. I allocate the remaining 30% thus it will finish in 12 days.

It would make the Gantt chart more flexible and in the case another task appears during the project it will show easily how changing the ressource breakdown affect all other tasks.

What do tou think about that aspect.
Is it possible?
Is there a work around?

Regards

**Comment by dbarashev** [comment]
Duplicates #83
